ou_dedetai/constants.py

```diff
@@ -31,11 +41,21 @@
 CACHE_LIFETIME_HOURS = 12
 """How long to wait before considering our version cache invalid"""
 
+if RUNMODE == 'snap':
+    cache_dir = Path(os.getenv('SNAP_USER_COMMON')) / '.cache'
+else:
+    try:
+        cache_dir = Path(os.getenv('XDG_CACHE_HOME'))
```

Comment:
Can this use getenv's default argument instead of a try/catch?

Reply:
I was thinking:
```
cache_dir = Path(os.getenv("XDG_CACHE_DIR", Path.home() / ".cache")
```

Reply:
of course. Right you are.